dropbox-sdk-android/src/main/java/com/dropbox/core/android/AuthActivity.kt

```diff
@@ -82,7 +82,7 @@ public open class AuthActivity : Activity() {
      * See:
      * https://developer.android.com/reference/android/app/Activity#onTopResumedActivityChanged(boolean)
      */
-    override fun onTopResumedActivityChanged(onTop: Boolean) {
+    fun onTopResumedActivityChanged(onTop: Boolean) {
```

Comment:
Had to do this because the Android JAR file we use as `compileOnly` is not aware of this API.